Cargo.toml

```diff
@@ -31,3 +32,7 @@ required-features = ["derive"]
 
 [badges]
 maintenance = { status = "actively-developed" }
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
Wired up `doc_cfg` in this PR. Let me know if you'd like it split out separately, but I think it's nice when you start having several features.

Rendered rustdoc:

<img width="1006" src="https://user-images.githubusercontent.com/797/92574546-12777680-f23c-11ea-8ce2-f5b6ab8574f3.png">

<img width="664" src="https://user-images.githubusercontent.com/797/92574560-173c2a80-f23c-11ea-87d2-b8a26d984e3e.png">